src/fluent-bit.c

```diff
@@ -126,6 +126,7 @@ static void flb_help(int rc, struct flb_config *config)
     print_opt("-v, --verbose", "increase logging verbosity (default: info)");
 #ifdef FLB_HAVE_TRACE
     print_opt("-vv", "trace mode (available)");
+    print_opt("-Z, --disable-trace", "disable tracing completely");
```

Comment:
this should be an option to `enable a disabled option by default`: `--chunk-trace`

Reply:
The semantics will be changed to disabled by default, with an option to enable it via a command line or the configuration file in the service section.